README.md

```diff
@@ -12,17 +13,22 @@
 
 ## ⚒️ Built With
 
-- 
-- 
-
+- BOOTSTRAP
+- CSS
+- JAVASCRIPT
+- GRID
+- FLEX
+- LINTERS (stylint and eslint)
 ## Live Demo
-- [Coming soon]
+:point_right: [g-summit](https://altontonn.github.io/Global-Summit/)
 
 ## 🧰 Tools
 
 The following tools help make it easier to run this code.
 
```

Comment:
- [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).